labs-ios-starter/Application/View/UI/Custom Objects/CircularButton.swift

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2020 Lambda, Inc. All rights reserved.
+// CircularButton.swift
+
+import UIKit
+
+/// The system icons used throughout the app, enums prevents typos -> UIImage(systemName: "[HERE]")
+enum Icon: String {
+    case plus = "plus"
+}
+
+/// A rounded button with an ICON
+class CircularButton: UIButton {
+    
+    // MARK: - Initializers
+    
+    /// The initializer called directly by the Developer
+    convenience init(withIcon systemName: Icon?, size: CGFloat?) {
+        self.init()
+        
+        if let systemName = systemName?.rawValue {
+            setImage(UIImage(systemName: systemName), for: .normal)
+        }
+        
+        if let size = size {
+            setDimensions(width: size, height: size)
+            layer.cornerRadius = size / 2 // Circular shape            
+        }
+    }
+    
+    /// Programmatic init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    /// Storyboard init
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+    
+    /// Sets up the button with configuration
+    private func commonInit() {
+        setShadow()
+        setColors()
+    }
+    
+    // MARK: - Configuration Methods
+    
+    /// Configure the appearance of the button's shadow
+    private func setShadow() {
+        layer.shadowColor = UIColor(red: 0.64, green: 0.71, blue: 0.78, alpha: 1.00).cgColor
```

Comment:
Do we always want the shadow to be a consistent color? Maybe we could use a color set here to allow for light/dark mode?